### DISCUSSION TITLE
New remote mode for quick access

### DISCUSSION THREAD

**Discussion by MrShinyBoots** [ChurchApps/ChurchAppsSupport#327] [FreeShow] [answered]
Hi there - I'm wondering about the possibility of adding a new remote mode - my use case is less church based and more general live usage - by adding lots of songs to a project I would love the ability to connect with a device say a phone and be able to search andselect from this list as you can currently with remote - however i would like 2 things to happen in this new connection type.

1. a search function so a song can be found quickly from a large list - this is already available in the shows tab on remote - however most other tabs would not be needed for this usage.

2. for the presentation to automatically load at the first slide when clicked on the shows tab

I am planning to use freeshow on a mini pc / raspberry pi, then use it for music events, band lyrics etc however i don't always want to pre design a full setlist. rather i would like to just enter songs into freeshow complete with chord charts etc then have a simple connection to browse the list and select the presentation with as few clicks as possible, once loaded the show would be controlled via midi footswitch. 

Ideally you would need a projects folder tab - so you could group by band if you wished and a shows tab with search that auto loads the first slide of a show, tis remote connection could then be used to quickly search and load any song directly to the main or stage display or both depending on what you have set up for that show. essentially all planning work could be done at home then on usage a simple connection would control the loading of a searchable song and a footswitch would control the next / previous slide / song.

so essentially a stripped down remote mode with the first slide of a show loading automatically, this way people could select a song from the list and load it quickly with minimal extra steps.

**Comment by vassbo** [comment]
Will see if this can be added in a way.

**Comment by vassbo** [comment, marked as answer]
In version 1.2.5 you will be able to enable "Quick play" in the RemoteShow shows list, this should behave like you mentioned here.
